src/node_util.cc

```diff
@@ -254,12 +254,59 @@ static void ParseEnv(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(dotenv.ToObject(env));
 }
 
+static void GetCallSite(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+
+  CHECK_EQ(args.Length(), 1);
+  CHECK(args[0]->IsNumber());
+  const uint32_t frames = args[0].As<Uint32>()->Value();
+
+  // +1 for disregarding node:util
+  Local<StackTrace> stack = StackTrace::CurrentStackTrace(isolate, frames + 1);
+  const int frame_count = stack->GetFrameCount();
+  std::vector<Local<Value>> callsite_objects{};
```

Comment:
better to use `v8::LocalVector<v8::Value>` here

Reply:
Done in [38f7b0b](https://github.com/nodejs/node/pull/54380/commits/38f7b0b10f26554616910e04d775d262c8d35fa4)